File: sdk/binding/android/q-voice-java/src/main/java/com/qdreamer/qvoice/listener/OnQVoiceCallback.java
package com.qdreamer.qvoice.listener;

public interface OnQVoiceCallback {

    /**
     * SDK 结果回调
     * @param type 消息类型 {@link com.qdreamer.qvoice.constants.QVoiceCode}
     * @param data 消息数据，data 可能为 null
     */
    void onQVoiceCallback(byte type, byte[] data);

}

File: sdk/binding/android/q-voice/src/main/java/com/qdreamer/qvoice/listener/OnQVoiceCallback.kt
package com.qdreamer.qvoice.listener

interface OnQVoiceCallback {

    /**
     * [type] 回调消息类型
     * [data] 对应消息类型的真实数据
     */
    fun onQVoiceCallback(type: Byte, data: ByteArray?)

}